pages/payment-incomplete/index.html

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: Payment Incomplete
+excerpt: We were unable to process your payment. Please ensure you've entered your payment information correctly, or reach out to <a href="mailto:sales@gruntwork.io">sales@gruntwork.io</a> if you need assistance.
+permalink: /payment-incomplete/
+slug: payment-incomplete
+---
+
+<div class="main">
+  <div class="section section-hero">
+    <div class="container">
+      <div class="row">
+        <div class="col-xs-12 text-center">
+          <h1>{{ page.title }}</h1>
+          <p>{{ page.excerpt }}</p>
+          <p>
```

Comment:
I think this extra `<p>` wrapper is unnecessary.

Reply:
✅